rel-eng/atomic-enterprise-master.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Atomic Enterprise Master
+Documentation=https://github.com/openshift/origin
+After=network.target
+After=etcd.service
+Before=atomic-enterpries-node.service
+Before=atomic-enterprise-sdn-node.service
+Before=atomic-enterprise-sdn-master.service
+Requires=network.target
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/sysconfig/atomic-enterprise-master
+Environment=GOTRACEBACK=crash
+ExecStart=/usr/bin/atomic-enterprise start master --config=${CONFIG_FILE} $OPTIONS
+LimitNOFILE=131072
+LimitCORE=infinity
+WorkingDirectory=/var/lib/origin/
```

Comment:
Yup. I've made the change but have not posted it up just yet as I was working on some other requests at the same time.